core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -49,10 +51,14 @@ oppia.directive('audioTranslationBar', [
       link: function(scope: ICustomScope, elm) {
         scope.getVoiceoverRecorder();
 
+        var userIsLoggedIn;
+        UserService.getUserInfoAsync().then(function(userInfo) {
+          userIsLoggedIn = userInfo.isLoggedIn();
```

Comment:
You can set `scope.userIsGuest` here directly.

Reply:
`scope.userIsGuest` is changed in some other part of the code. I think it may affect its value depending on which part gets run first

Reply:
I see now.